### PR TITLE
Opower: Fix link to the energy dashboard configuration

### PR DESCRIPTION
**Pull request by tronikos**
## Proposed change

SSIA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/opower.markdown [hunk 1]
```diff
@@ -60,7 +60,7 @@ For gas:
 
 Because utilities only release usage/cost data with a 48-hour delay, the integration inserts data into statistic objects. At the initial setup, the integration pulls historical monthly usage/cost since the account activation, daily usage/cost for the past 3 years, and for electricity hourly usage/cost for the past 2 months. After the initial setup, the integration keeps pulling hourly data for electricity and daily data for gas for the past 30 days to allow for any corrections in the data from the utilities.
 
-In the configuration of the energy dashboard (**{% my energy title="Settings > Dashboards > Energy" %}**):
+In the configuration of the energy dashboard (**{% my config/energy title="Settings > Dashboards > Energy" %}**):
```
Comment (frenck) [inline source/_integrations/opower.markdown]: `config/energy` isn't valid, as they cannot contain `/`.

According to:
https://my.home-assistant.io/create-link/?redirect=energy

The existing content is correct.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/opower.markdown (hunk 1 above)
Comment (tronikos) [reply source/_integrations/opower.markdown]: The problem is it redirects to the energy dashboard and not to its configuration where you need to follow the steps below. Maybe remove the my link like I originally had in https://github.com/home-assistant/home-assistant.io/pull/26793 ?

Comment (frenck) [reply source/_integrations/opower.markdown]: Aah I see:

`config_energy` is the right key in that case.

Comment (tronikos) [reply source/_integrations/opower.markdown]: Thanks! PTAL

**Review by frenck — APPROVED** [review]
Thanks, @tronikos 👍 

../Frenck

**Pull request merged**